src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -87,6 +88,10 @@ public void setSoTimeout(int soTimeout) {
     this.soTimeout = soTimeout;
   }
 
+  public void setListener(JedisCommandListener listener) {
```

Comment:
should be addListener?

Reply:
The new multi-listener interface contains:  `setListeners`, `addListener`, and `removeListener`